torchgeo/datasets/cropharvest.py

```diff
@@ -0,0 +1,323 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""CropHarvest datasets."""
+
+import glob
+import json
+import os
+from typing import Callable, Optional
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import torch
+from matplotlib.figure import Figure
+from torch import Tensor
+
+from .geo import NonGeoDataset
+from .utils import (
+    DatasetNotFoundError,
+    download_url,
+    extract_archive,
+    percentile_normalization,
+)
+
+
+class CropHarvest(NonGeoDataset):
+    """CropHarvest dataset.
+
+    `CropHarvest <https://github.com/nasaharvest/cropharvest>`_ is a
```

Comment:
```suggestion
    `CropHarvest <https://github.com/nasaharvest/cropharvest>`__ is a
```
rST is the worst file format ever created...

Double underscore needed so that the link in the table points to the class docs instead of to GitHub.